pkg/cmd/server/server.go

```diff
@@ -163,6 +164,7 @@ func NewCommand(f client.Factory) *cobra.Command {
 			defaultVolumesToFsBackup:       podvolume.DefaultVolumesToFsBackup,
 			uploaderType:                   uploader.ResticType,
 			maxConcurrentK8SConnections:    defaultMaxConcurrentK8SConnections,
+			useInformerCacheForGet:         true,
```

Comment:
```suggestion
			useInformerCacheForGet:         defaultUseInformerCacheForGet,
```

Can we make this a variable like above so all defaults are next to each other?

Reply:
Sure, easy enough. Will update.